__tests__/app.test.js

```diff
@@ -58,7 +58,7 @@ describe("/api/articles/:article_id", () => {
           expect(article.article_id).toBe(1);
         });
     });
-    test("status: 200 - article object in response has these properties: author, title, article_id, body, topic, created_at, votes", () => {
+    test("status: 200 - article object in response has these properties: author, title, article_id, body, topic, created_at, votes, comment_count", () => {
```

Comment:
This is fine but worth noting that when adding features later that alter response objects it can be a good idea to create a new test rather than amend previous ones to have the property.